apps/meteor/server/models/raw/Statistics.ts

```diff
@@ -26,6 +26,22 @@ export class StatisticsRaw extends BaseRaw<IStats> implements IStatisticsModel {
 		return records?.[0];
 	}
 
+	async findLastStatsToken(): Promise<IStats['statsToken']> {
+		const records = await this.find(
```

Comment:
Can we use a `findOne` here? it's technically the same, but this is what findOne is for 👀

Reply:
I just copied the one above, do you think it is ok if I do this another time? I'll make sure to track it so I don't forget. Just so we can merge this asap

Reply:
No probs with that.